<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网关登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/ico/ico.ico</normaloff>:/ico/ico.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;
border-radius:10px;
padding:2px 4px;
background-color: rgba(255, 255, 255,0);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="userNameLabel">
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameText">
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgba(255, 255, 255,166);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="remUser">
          <property name="text">
           <string>记住账号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pswodLabel">
            <property name="text">
             <string>密码  ：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordText">
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color: rgba(255, 255, 255,166);</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rempswd">
            <property name="text">
             <string>记住密码</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,166);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuBar
{
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
background-color: rgba(255, 255, 255,0);
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loginClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginClick()</slot>
 </slots>
</ui>
